ansible/roles/products/teleop/server/desktop-icons/templates/scripts/start-docker-container-nuc.j2

```diff
@@ -21,6 +21,6 @@ docker start ${CONTAINER}
 echo "Starting the container..."
 sleep 3
 docker exec --user user ${CONTAINER} bash -c "source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;roscd sr_ethercat_hand_config/launch;git checkout sr_rhand.launch;git checkout sr_lhand.launch;git checkout ${SR_BRANCH}"
-docker exec --user user ${CONTAINER} bash -c "source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;ulimit -c unlimited;roslaunch sr_teleop_launch hardware_control.launch hand_serial:=${HAND_SERIALN} eth_port:=${ETH_PORT} side:=${HAND_SIDE} robot_model:=${UR_TYPE} 2> >(tee -a /home/user/.ros/log/stderr.log >&2)"
+docker exec --user user ${CONTAINER} bash -c "source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash;ulimit -c unlimited;roslaunch {{ teleop_control_launch }} hand_serial:=${HAND_SERIALN} eth_port:=${ETH_PORT} side:=${HAND_SIDE} robot_model:=${UR_TYPE} biotacs:={{ biotacs }} 2> >(tee -a /home/user/.ros/log/stderr.log >&2)"
```

Comment:
Dont you need to pass down the biotacs var?

Reply:
Fixed in the last commit